youtube_dl/extractor/funimation.py

```diff
@@ -227,6 +249,81 @@ def _real_extract(self, url):
             'thumbnail': thumbnail,
             'formats': formats,
         }
+
+
+class FunimationShowPlaylistIE(FunimationBaseIE):
+    IE_NAME = 'funimation:playlist'
+    _VALID_URL = r'(https?://(?:www\.)?funimation\.com/shows/(?P<id>[^/]+))(?:/(?:home|about|videos))?$'
+    _TESTS = [{
+        'url': 'http://www.funimation.com/shows/a-certain-scientific-railgun/home',
+        'info_dict': {
+            'id': 'a-certain-scientific-railgun',
+            'description': '	Misaka’s electro-manipulation abilities – and delightfully destructive Railgun projectile move – make her a rock star in Academy City. The techno-metropolis is packed with supernaturally powered students known as espers, including Misaka’s flirty friend and roommate, Kuroko. She uses her teleportation skills as a member of the Judgment law enforcement team, fighting crime alongside her fellow agent Uiharu. Joined by their friend Saten, a spunky Level 0 esper, Misaka,',
+            'title': 'A Certain Scientific Railgun'
+        },
+        'playlist_count': 48
+    }, {
+        'url': 'http://www.funimation.com/shows/hacksign/home',
+        'info_dict': {
+            'id': 'hacksign',
+            'description': '	Tsukasa wakes up inside The World, a massive online role-playing game full of magic and monsters, and finds himself unable to log out. With no knowledge of what’s happening in the real world, Tsukasa must discover how he ended up stuck in the game, and what connection he has with the fabled Key of the Twilight—an item that’s rumored to grant ultimate control over the digital realm.',
+            'title': '.hack//SIGN'
+        },
+        'playlist_count': 56
+    }]
+
+    def _real_extract(self, url):
+        display_id = self._match_id(url)
+
+        user_agent = self._extract_cloudflare_session_ua(url)
+
+        # Use series page to get ID number and title / description
+        series_url = self._search_regex(self._VALID_URL, url, 'series URL')
+        request = sanitized_Request(series_url)
+        request.add_header('User-Agent', user_agent)
+        webpage = self._download_webpage(request, display_id, 'Downloading series webpage')
+
+        # Parseable show data stored as a JavaScript variable
+        playlist = self._parse_json(
+            self._search_regex(
+                r'var\s+playersData\s*=\s*(\[.+?\]);\n',
+                webpage, 'players data'),
+            display_id)[0]['playlist'][0]
+
+        def pagefunc(pagenum):
+            # Internal Funimation endpoint for getting paginated video list HTML
+            request = sanitized_Request(
+                'https://www.funimation.com/shows/viewAllFiltered?section=episodes&showid={}&offset={}'
+                .format(playlist.get('showId'), pagenum * 20))
+            request.add_header('User-Agent', user_agent)
+            episode_list = self._download_json(
+                request, display_id, 'Downloading episode list from {}'.format(pagenum * 20))['main']
```

Comment:
Compatibility kicks me in the ass again. Will fix quickly. (And sorry for not testing 2.6. I forgot on the last commit.)